packages/serverless/settings/observability_project/index.ts

```diff
@@ -27,4 +27,5 @@ export const OBSERVABILITY_PROJECT_SETTINGS = [
   settings.OBSERVABILITY_APM_TRACE_EXPLORER_TAB_ID,
   settings.OBSERVABILITY_ENABLE_AWS_LAMBDA_METRICS_ID,
   settings.OBSERVABILITY_APM_ENABLE_CRITICAL_PATH_ID,
+  settings.OBSERVABILITY_ENABLE_INFRASTRUCTURE_HOSTS_VIEW_ID,
```

Comment:
Hi @alisonelizabeth I reached out to understand further but I was under the impression that there wasn't going to be an observability advanced settings area or at least one where Infra could have a setting.  If there is and we can keep our setting there, we don't necessarily need to add it to our own UI.

Reply:
@mykolaharmash I think we need to leave this out for now and add it when we "flip the switch" to enable our plugin.  Otherwise this setting will show up with no Infra UI if we haven't enabled the plugin for serverless yet.

Reply:
> Hi @alisonelizabeth I reached out to understand further but I was under the impression that there wasn't going to be an observability advanced settings area or at least one where Infra could have a setting. If there is and we can keep our setting there, we don't necessarily need to add it to our own UI. Since it won't work in our UI without enabling it here, I assume we then have to show it in the Project Advanced Settings and I misinterpreted the docs.

You can go to `/app/management/kibana/settings` to view Advanced Settings in serverless for each project type.

It is possible to have project-specific settings. The settings are defined [here](https://github.com/elastic/kibana/tree/main/packages/serverless/settings). You'll see that there is a set of common settings that are used across project types, as well as project-specific settings.

Reply:
> FYI - this setting will also appear on the Advanced Settings page in serverless. Just pointing out as it wasn't mentioned in the PR description.

Thank you @alisonelizabeth for pointing this out! Somehow I thought that the allowlist only enables the changes through the API and does not affect the UI 🙈

Reply:
> @mykolaharmash I think we need to leave this out for now and add it when we "flip the switch" to enable our plugin. Otherwise this setting will show up with no Infra UI if we haven't enabled the plugin for serverless yet.

Agree 💯